### PR TITLE
Add discourse example in setup.cfg

### PR DESCRIPTION
**Pull request by Nolski**
# Overview

As per other data sources, it may be useful for newcomers to have a commented out configuration for the discourse backend.

This configuration information was given in [issue #132 of grimoirelab-tutorial](https://github.com/chaoss/grimoirelab-tutorial/issues/132)

### CODE REVIEW

File: default-grimoirelab-settings/setup.cfg [hunk 1]
```diff
@@ -57,6 +57,11 @@ panels = true
 #raw_index = confluence_demo_raw
 #enriched_index = confluence_demo_enriched
 
+#[discourse]
+#raw_index = discourse_raw
+#enriched_index = discourse_enriched
+#no-archive = true
```
Comment (zhquan) [inline default-grimoirelab-settings/setup.cfg:63]: Could you follow the same naming format?
- discourse_demo_raw
- discourse_demo_enriched

Comment (Nolski) [reply default-grimoirelab-settings/setup.cfg:63]: Ah yes, that would make sense. Please see my latest commit for these updates

**Comment by zhquan** [comment]
And the last thing. Could you squash your commits into one single commit?

**Comment by Nolski** [comment]
@zhquan Just squashed my commits and pushed.

**Comment by zhquan** [comment]
@Nolski LGTM thank you very much for your PR

**Pull request merged**